app/src/main/java/com/example/mytrainingpal/components/EnterPainPrompt.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.mytrainingpal.components
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Edit
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun EnterPainPrompt(navigateToMusclePain: () -> Unit) {
+
+// Card Reference: https://foso.github.io/Jetpack-Compose-Playground/material/card/
+    Card(
+        border = BorderStroke(2.dp, MaterialTheme.colors.secondary),
+        backgroundColor = MaterialTheme.colors.primary,
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(10.dp),
+        elevation = 10.dp,
+
+    ){
```

Comment:
There is already an existing card widget for this called `WidgetCard`. Could you please use this? This makes all cards in our app look more consistent. e.g. your card here does not have round corners. `WidgetCard` is already used in the code. There you can see, how to use it.

In order to have an orange border color, you need to add an additional parameter to `WidgetCard` that toggles whether it should have an orange border or not.

Reply:
Thanks Micha! :)
WidgetCard() now accepts a boolean parameter that defines the border color, therefore I had to add arguments to all composables that use WidgetCard().

EnterPainPrompt() is now a WidgetCard().